_posts/2018-08-01-mentor-spotlight-pablo-aguiar.markdown

```diff
@@ -0,0 +1,44 @@
+
+### Tell us about yourself. What is your background and what do you do now?
+Hi! I’m a Computer Scientist from Brazil. In the past few years I had a lot of fun being a software engineer at Globo.com, Latin America’s largest media and publishing company. Nowadays I live in Munich, Germany, working with Electro-Mobility at The Mobility House and having fun with swarms of batteries and the Internet of Charging, reacting to the european electricity grid demands in real-time.
+
+### How did you get started in tech?
+I started programing computers at the age of 11 with lessons from my late father. He first taught me Pascal and, once I was familiar with a few algorithms and data structures, we moved to C. We had and agreement that I only could play computer games once I’ve completed my programming lessons. I’m truly grateful to him for this. Since then I never stopped programming and grew up to become a passionate Full Stack Software Engineer.
+What are some of the projects you are working on currently?
+Currently, I’m working on a distributed Go application that is deployed to a multitude of 3G routers, each connected to a battery module. Based on instructions sent via a message broker, the application talks to the battery module via modbus and smartly controls the battery’s charge and discharge functions, helping maintain the electricity grid stable.
+I also work on some open source projects such as [TheFuck](https://github.com/nvbn/thefuck), [Passpie](https://github.com/marcwebbie/passpie), [Oh My Fish!](https://github.com/oh-my-fish), [Tsuru](https://github.com/tsuru), [Thumbor](https://github.com/thumbor), [Elastalert](https://github.com/Yelp/elastalert), etc., either for fun or to make them better or to fix any issue or add a feature that suits my needs.
+
+### What is open source/FOSS to you?
+For me FOSS means so many things. It is an opportunity to learn from exceptional software engineers. It is an increase in security and stability gained from peer reviews. It’s freedom to control the software, to customize it and to build on top of what’s already achieved while still being flexible. And ultimately it is an opportunity to share, to collaborate with others and to give back, in a transparent way. I also like the idea of playing an active role in improving the world, which is possible only when everyone has access to the way that world is designed.
+
+### Why do you contribute to open source?
+I contribute to open source because I really love to code. It also makes me a better engineer as I learn so much in the process, constantly. Another reason is that I believe in the open source culture and I see value in helping others and being helped.
+What was your first interaction/impression of open source and how has your involvement in open source evolved over time?
```

Comment:
Should this bolded as well, or is the purpose here to kind of have a sub question?